Datasets/LanguageModelDataset.swift

```diff
@@ -105,28 +105,12 @@ extension LanguageModelDataset: Collection {
   }
 }
 
-//Extension when Item is [Int] and openItem is not needed
-extension LanguageModelDataset where Item == [Int] {
-  public init(batchSize: Int, sequenceLength: Int, items: [Item], lengths: [Int]) {
-    self.init(
-      openItem: { $0 },
-      batchSize: batchSize,
-      sequenceLength: sequenceLength,
-      items: items,
-      lengths: lengths)
-  }
-
-  public init(batchSize: Int, sequenceLength: Int, items: [Item]) {
-    self.init(
-      openItem: { $0 },
-      batchSize: batchSize,
-      sequenceLength: sequenceLength,
-      items: items,
-      lengths: items.map { $0.count })
-  }
-}
-
-//sampleIndices function to use in conjunction with a LanguageModelDataset
+/// The sampleIndices function to use in conjunction with a `LanguageModelDataset` in a `Batcher`.
+/// Will shuffle the dataset in place instead of the indices (like the default function does).
+/// - Parameters:
+///   - dataset: The underlying `LanguageModelDataset`.
+///   - shuffled: Shuffle or not the dataset.
```

Comment:
Should `shuffled` be named `shuffle`? The boolean flag sounds like it should be used as an imperative verb.
```suggestion
///   - shuffled: Shuffles the data iff `true`.
```

Reply:
That would be a broader change in `Batcher`. I have no strong opinion on either syntax.